docs/BreakingChanges/050 Use simpler 4.0 SQL generation in Entity Framework.md

```diff
@@ -0,0 +1,27 @@
+## 50: Opt-in break to revert from different 4.5 SQL generation to simpler 4.0 SQL generation
```

Comment:
{
  "Id": "50",
  "Title": "Opt-in break to revert from different 4.5 SQL generation to simpler 4.0 SQL generation",
  "Details": "Queries that produce JOIN statements and contain a call to a limiting operation without first using OrderBy now produce simpler SQL. After upgrading to .NET Framework 4.5, these queries produced more complicated SQL than previous versions.",
  "Suggestion": "This feature is disabled by default. If Entity Framework generates extra JOIN statements that cause performance degradation, you can enable this feature by adding the following entry to the <appSettings> section of the application configuration (app.config) file:\n\n```xml\n<add key=\"EntityFramework_SimplifyLimitOperations\" value=\"true\" />\n```",
  "Link": "https://msdn.microsoft.com/en-us/library/dn720772(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.2",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis